app/notification.go

```diff
@@ -639,6 +639,11 @@ func (a *App) userAllowsEmail(user *model.User, channelMemberNotificationProps m
 		}
 	}
 
+	// if user is a bot account, then we do not send email
+	if user.IsBot {
```

Comment:
Should we check this in the beginning of the method?

Reply:
Hmm good point. I will move this to the start of the method!